bpython/autocomplete.py

```diff
@@ -335,6 +335,48 @@ def list_attributes(self, obj):
                 return dir(obj)
 
 
+class ArrayItemMembersCompletion(BaseCompletionType):
+
+    def __init__(self, shown_before_tab=True, mode=SIMPLE):
+        self._shown_before_tab = shown_before_tab
+        self.completers = CumulativeCompleter((
+                            StringLiteralAttrCompletion(mode=mode),
```

Comment:
Good catch. Yeah I don't think that is necessary any more. I was using it in a previous version of this code, but no longer need it for the "temp_val_from_array" logic I am now using. I'll remove all but AttrCompletion.